rispy/parser.py

```diff
@@ -204,29 +215,34 @@ def _add_list_value(self, record: dict, name: str, value: Union[str, list[str]])
             must_exist = record[name]
             record[name] = [must_exist, *value_list]
 
-    def _add_tag(
-        self, record: dict, tag: str, content: str, extend_multiline: bool = False
-    ) -> None:
+    def _extend_tag(self, record: dict, tag: str, content: Union[str, list[str]]) -> None:
+        """Extend tags with multiline values."""
+
+        sep = " " if self.undo_wrapping else "\n"
+
+        name = self.mapping[tag]
+        if isinstance(record[name], list):
+            record[name][-1] = sep.join((record[name][-1], content))
+        else:
+            record[name] = sep.join((record[name], content))
+
+    def _add_tag(self, record: dict, tag: str, content: str) -> None:
         try:
             name = self.mapping[tag]
         except KeyError:
             if self.skip_unknown_tags:
                 return
 
-            # handle unknown tag
-            name = self.mapping[self.UNKNOWN_TAG]
-            if name not in record:
-                record[name] = defaultdict(list)
-            record[name][tag].append(content)
-
+            record.setdefault("unknown_tag", defaultdict(list))[tag].append(content)
```

Comment:
since we removed `UK`, make `unknown_tag` a variable constant UNKNOWN_TAG since right now it's a magic string used in a few places